src/controller/content/content.service.ts

```diff
@@ -945,4 +933,63 @@ export class ContentService {
       throw error;
     }
   }
+ 
+  async updateOpenEndedQuestion(questionId:number,openEndedBody: UpdateOpenEndedDto)
+  {
+    try{
+      await db
+      .update(zuvyOpenEndedQuestion)
+      .set(openEndedBody)
+      .where(eq(zuvyOpenEndedQuestion.id, questionId));
+
+      return {
+        status: 'success',
+        code: 200,
+        message: 'Open ended question has been updated successfully'
+      }
+    }
+    catch(err)
+    {
+      throw err;
+    }
+  }
+
+  async deleteQuiz(id: deleteQuestionDto) {
+    try {
+      await db.delete(zuvyModuleQuiz).where(sql`${inArray(zuvyModuleQuiz.id,id.questionIds)}`);
+      return {
+        status: 'success',
+        code: 200,
+        message: 'The quiz questions has been deleted successfully',
+      };
+    } catch (err) {
+      throw err;
+    }
+  }
+
+  async deleteCodingProblem(id: deleteQuestionDto) {
+    try {
+      await db.delete(zuvyCodingQuestions).where(sql`${inArray(zuvyCodingQuestions.id,id.questionIds)}`);
```

Comment:
@Arunesh, we need to eliminate references to these collections from other tables. Additionally, there might be some other edge cases to address.
@Souvikdeb2612